File: Iniciante/1094/1094.c
#include <stdio.h>

int main() {
  int N=0, Quantia;
  int coelhos=0, ratos=0, sapos=0, total=0;
  int i;
  float percentual;
  char Tipo;

  scanf("%d",&N);

  for(i=0; i<N; i++){
    scanf("%d %c",&Quantia,&Tipo);

    if(Tipo == 'C'){
      coelhos = coelhos + Quantia;
    }

    if(Tipo == 'R'){
      ratos = ratos + Quantia;
    }

    if(Tipo == 'S'){
      sapos = sapos + Quantia;
    }
  }

  total = coelhos+ratos+sapos;

  printf("Total: %d cobaias\n",total);
  printf("Total de coelhos: %d\n",coelhos);
  printf("Total de ratos: %d\n",ratos);
  printf("Total de sapos: %d\n",sapos);

  percentual = (coelhos*100.00)/total;
  printf("Percentual de coelhos: %.2f %%\n",percentual);

  percentual = (ratos*100.00)/total;
  printf("Percentual de ratos: %.2f %%\n",percentual);
  
  percentual = (sapos*100.00)/total;
  printf("Percentual de sapos: %.2f %%\n",percentual);

  return 0;

}

File: Python/Iniciante/1094.py
# Problema 1094 - Beecrowd - Iniciante - Nível 3

# Iniciamos com todos os resultados como 0
coelhos = ratos = sapos = total = 0

# Recebemos do usuário o número de entradas que iremos ter,
# inserindo como o número de iterações do FOR
for i in range(int(input())):

    # A cada iteração, separamos a 'quantidade' do 'animal'
    # somamos ao total de animais
    quantidade, animal = input().split(' ')
    total += int(quantidade)

    # Adicionamos também o valor ao respectivo resultado do 'animal'
    if animal == 'C':
        coelhos += int(quantidade)
    elif animal == 'R':
        ratos += int(quantidade)
    elif animal == 'S':
        sapos += int(quantidade)

# Após o final do FOR e de suas iterações, mostramos os resultados
# de acordo com a formatação desejada.
print(f"Total: {total} cobaias")
print(f"Total de coelhos: {coelhos}")
print(f"Total de ratos: {ratos}")
print(f"Total de sapos: {sapos}")
print(f"Percentual de coelhos: {((100.0 / total) * coelhos):.2f} %")
print(f"Percentual de ratos: {((100.0 / total) * ratos):.2f} %")
print(f"Percentual de sapos: {((100.0 / total) * sapos):.2f} %")